deker/validators.py

```diff
@@ -134,6 +134,46 @@ def process_attributes(
     return primary_attributes, custom_attributes
 
 
+def validate_custom_attributes(
+    schema: Union["ArraySchema", "VArraySchema"],
+    dimensions: Tuple[Union[Dimension, TimeDimension], ...],
+    primary_attributes: dict,
+    custom_attributes: dict,
+    attributes: Optional[dict],
+) -> dict:
+    """Validate custom attributes over schema.
+
+    :param schema: ArraySchema or VArraySchema instance
+    :param dimensions: tuple of (V)Array dimensions
+    :param primary_attributes: (V)Array primary attributes
+    :param custom_attributes: old custom attributes
+    :param attributes: new custom attributes to validate
+    """
+    from deker.schemas import TimeDimensionSchema
```

Comment:
Why import it here and not in the start?

Reply:
Circular Import